hnn_qt5.py

```diff
@@ -62,7 +62,11 @@ def parseargs ():
 debug = dconf['debug']
 testLFP = dconf['testlfp'] or dconf['testlaminarlfp']
 
-defncore = multiprocessing.cpu_count() # default number of cores
+# get default number of cores
+try:
+  defncore = len(os.sched_getaffinity(0))
+except AttributeError:
+  defncore = 4
```

Comment:
This needs to be fixed. Left by mistake

Reply:
Mistake is setting to 4 cores by default?